### ISSUE TITLE
[nsfw] przekieruj.dhlparcel.pl: breakage

### ISSUE THREAD

**Issue by rafnov** [uBlockOrigin/uAssets#24785]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [X] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://przekieruj.dhlparcel.pl/pl/disposition/index`

### Category

breakage

### Description

No checkbox to accept rules on waterfox with uBlock

### Other extensions used

Dark reader

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

https://ibb.co/7S0442Z
https://ibb.co/1q4MCZj

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.59.0
Firefox Mobile: 127
filterset (summary):
 network: 158531
 cosmetic: 64318
 scriptlet: 20863
 html: 1912
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  ublock-filters: 40168-120, now
  ublock-badware: 10213-1, now
  ublock-privacy: 1040-4, now
  ublock-unbreak: 2530-1, now
  ublock-quick-fixes: 142-2, now
  adguard-mobile: 9638-74, now
  easylist: 87841-241, now
  easyprivacy: 53073-69, now
  urlhaus-1: 29210-0, 32m
  plowe-0: 3548-1063, now
  POL-0: 8762-46, 32m
  POL-2: 1283-48, now
filterset (user): [empty]
trustedset:
 added: [array of 2 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 1221 ms (selfie)
 maxAssetCacheWait: 185 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 1
 network:
  googletagmanager.com: 1
 extended:
  ##+js(set-constant, rwt, noopFunc)
  ##+js(href-sanitizer, #main a[href^="/url?q=http"], ?q)
```
</details>

**Comment by garry-ut99** [comment]
> ### Prerequisites

>    * [x]  I have verified that other extensions are not causing the issue.

You didn't, the checkbox is here, just is invisible because of black color caused by your Dark Reader extension.

>    * [x]  I did not answer truthfully to **ALL** the above checkboxes.

Yes, you didn't

Spammers are not welcome here.

**Comment by rafnov** [comment]
You are right, sorry, my mistake, I just haven't seen the issue with checkbox before.
